tests/templates/kuttl/external-access/opensearch.yaml.j2

```diff
@@ -60,16 +60,16 @@ spec:
                       annotations:
                         secrets.stackable.tech/scope: node,pod,service=opensearch-nodes-data2-headless
     envOverrides:
-      # TODO Make these the defaults in the image
+      # Only required for the official image
+      # The official image (built with https://github.com/opensearch-project/opensearch-build)
+      # installs a demo configuration if not disabled explicitly.
       DISABLE_INSTALL_DEMO_CONFIG: "true"
       OPENSEARCH_HOME: {{ test_scenario['values']['opensearch_home'] }}
     configOverrides:
-      # TODO Add the required options to the operator
       opensearch.yml:
         # Disable memory mapping in this test; If memory mapping were activated, the kernel setting
         # vm.max_map_count would have to be increased to 262144 on the node.
         node.store.allow_mmap: "false"
-        # TODO Check that this is safe despite the warning in the documentation
```

Comment:
It is safe if the demo configuration is replaced with `podOverrides`. To be on the safe side, I removed the demo configuration in stackabletech/docker-images#1228.